Formula/poac.rb

```diff
@@ -0,0 +1,33 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm/poac"
+  url "https://github.com/poacpm/poac.git",
+    tag:      "0.3.1",
+    revision: "0afee72936c8c52ab2a477f0f14b45282bae83c1"
+  license "Apache-2.0"
+  head "https://github.com/poacpm/poac.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on "boost"
+  depends_on :macos
```

Comment:
Why does it depend on macOS and xcode?

Reply:
@SMillerDev 
This application cannot be built on Ubuntu 16.04 & macOS Catalina or earlier.
Should I remove them?

Reply:
Why not?

Reply:
@SMillerDev C++20 is required

Reply:
So then it can just depend on gcc on Linux and require a specific macOS version.

Reply:
@SMillerDev 

For Linux, I think I can use the following:

```ruby
on_linux do
  depends_on "gcc"
end
```
(Is this correct?)

But, could you please tell me how to require a specific macOS version?

Reply:
```rb
depends_on macos: :catalina # C++20
```

Reply:
Thank you!